### PR TITLE
Classify - Fix string loader tests

### PR DESCRIPTION
**Pull request by lucasrcezimbra**

### CODE REVIEW

File: tests/unit/pypyr/dsl_test.py [hunk 1]
```diff
@@ -3222,6 +3223,23 @@ def test_save_error_multiple_call(mocked_moduleloader):
 
 
 # endregion Step: save_error
+# region Step: __eq__
+
+@patch('pypyr.moduleloader.get_module')
+def test_step_eq_another_type(mocked_moduleloader):
```
Comment (yaythomas) [inline tests/unit/pypyr/dsl_test.py:3229]: ❤️

File: tests/unit/pypyr/loaders/string__test.py
Comment (yaythomas) [inline tests/unit/pypyr/loaders/string__test.py:1]: just noticed `string__test.py`, rather than `string_test.py` 😆

**Review by yaythomas — APPROVED** [review]
thank you so much!

File: tests/unit/pypyr/loaders/string__test.py
Comment (lucasrcezimbra) [reply tests/unit/pypyr/loaders/string__test.py:1]: It was on purpose because pytest doesn't accept two test files with the same name :confused:

Comment (yaythomas) [reply tests/unit/pypyr/loaders/string__test.py:1]: oh, of course - forgot about that little detail! 😆

**Pull request merged**